src/openloco/audio/audio.cpp

```diff
@@ -777,11 +778,75 @@ namespace openloco::audio
         return false;
     }
 
-    static void sub_48A274(vehicle* v)
+    static void sub_48A274(vehicle* baseVehicle)
     {
-        registers regs;
-        regs.esi = (int32_t)v;
-        call(0x0048A274, regs);
+        // TODO: move vehicle_26 cast up
+        auto v = baseVehicle->as_vehicle_2or6();
+        if (v == nullptr)
+            return;
+
+        if (v->sound_id == 0xFF)
+            return;
+
+        // TODO: left or top?
+        if (v->sprite_left == (int16_t)0x8000u)
```

Comment:
This will be like rct2 0x8000 indicates the sprite is off map like when you pickup a peep.

Reply:
Yeah I figured. I'm just not sure if the code uses `sprite_left` or `sprite_top`, since they both get stored into a variable, and I have issues understand high/low nibbles.